templateMaker/src/recoDefinitions.cpp

```diff
@@ -13,6 +13,7 @@ RNode recoDefinitions::run(RNode d)
                   .Define("Mu1_pt", getFromIdx, {"Muon_pt", "Idx_mu1"})
                   .Define("Mu1_sip3d", getFromIdx, {"Muon_sip3d", "Idx_mu1"})
                   .Define("Mu1_dxy", getFromIdx, {"Muon_dxy", "Idx_mu1"})
+                  .Define("Mu1_hasTriggerMatch", getIntFromIdx, {"Muon_hasTriggerMatch", "Idx_mu1"})
```

Comment:
please remove this line since it will break backward compatibility with old sample

Reply:
Idx_mu1 is outdated!

Reply:
This will be done in a separate PR when the W Sequence is defined.